packages/updatePrescriptionStatus/src/validation/content.ts

```diff
@@ -113,15 +122,31 @@ export function businessStatus(task: Task): string | undefined {
 
 export function statuses(task: Task): string | undefined {
   const status = task.status
-  if (status === "completed") {
-    const businessStatus: CodeableConcept | undefined = task.businessStatus
-    if (businessStatus) {
-      const coding: Coding = businessStatus.coding![0]
-      const code = coding.code
-      if (code && ["with pharmacy", "ready to collect"].includes(code.toLowerCase())) {
-        return "Completed state indicated for a prescription status requiring patient action."
-      }
-    }
+  const businessStatus: CodeableConcept | undefined = task.businessStatus
+
+  if (!businessStatus) {
+    return
+  }
+
+  const coding: Coding = businessStatus.coding![0]
+  const code = coding.code
+
+  if (!code) {
+    return
```

Comment:
I think we agreed in elab that we should ensure that the businessStatus is one that we expect.

The businessStatus, coding and code are all marked as required in the spec. So, we want to return a message if they don't exist. That is something I should have covered earlier, sorry.

Reply:
If the businessStatus, coding, or code field is missing, the following message is returned.

```
{
    "resourceType": "Bundle",
    "type": "transaction-response",
    "entry": [
        {
            "response": {
                "status": "400 Bad Request",
                "outcome": {
                    "resourceType": "OperationOutcome",
                    "meta": {
                        "lastUpdated": "2024-05-30T07:37:41.403Z"
                    },
                    "issue": [
                        {
                            "code": "processing",
                            "severity": "error",
                            "diagnostics": "Missing required field(s) - Status."
                        }
                    ]
                }
            },
            "fullUrl": "urn:uuid:4d70678c-81e4-4ff4-8c67-17596fd0aa46"
        }
    ]
}
```